worklist/general_utility_functions.py

```diff
@@ -0,0 +1,16 @@
+from collections import defaultdict
```

Comment:
Since (I think) we expect this file to grow a bit, I'd still rename this to, say, `wikipedia_utils.py`, and this is where we'd keep convenience wrappers for using the Wikipedia API.

Reply:
Makes sense. Done!